src/pypromice/process/L1toL2.py

```diff
@@ -45,15 +49,26 @@ def toL2(L1, T_0=273.15, ews=1013.246, ei0=6.1071, eps_overcast=1.,
     except Exception as e: 
         print('Flagging and fixing failed:')
         print(e)
+        
+    #stid = ds.station_id
+    
     
+    
+    ds = differenceQC(ds)                                                      # Flag and Remove difference outliers     
+    ds = percentileQC(ds)                                                      # Flag and remove percentile outliers
+
     T_100 = _getTempK(T_0)  
     ds['rh_u_cor'] = correctHumidity(ds['rh_u'], ds['t_u'],  
                                      T_0, T_100, ews, ei0)                       
         
-    # Determiune cloud cover
-    cc = calcCloudCoverage(ds['t_u'], T_0, eps_overcast, eps_clear,                  # Calculate cloud coverage
-                           ds['dlr'], ds.attrs['station_id'])  
-    ds['cc'] = (('time'), cc.data)
+    # Determiune cloud cover for on-ice stations
```

Comment:
Wondering why this change for cloud cover is here? This has nothing to do with percentiles I assume?

Reply:
This is something Baptiste implemented in the meantime, so no, nothing to do with the qc.